agent/connect/ca/provider_vault.go

```diff
@@ -582,6 +589,7 @@ func (v *VaultProvider) Stop() {
 func (v *VaultProvider) PrimaryUsesIntermediate() {}
 
 func ParseVaultCAConfig(raw map[string]interface{}) (*structs.VaultCAProviderConfig, error) {
+	fmt.Fprintf(os.Stderr, "%+v\n", raw)
```

Comment:
this may also be a silly question but why this addition and line 611?

Reply:
Sorry, I was using this to test and forgot to remove it